flight/targets/sprf3e/fw/Makefile

```diff
@@ -50,13 +50,13 @@ OPTMODULES += AltitudeHold
 OPTMODULES += PathPlanner
 OPTMODULES += VtolPathFollower
 OPTMODULES += FixedWingPathFollower
-OPTMODULES += UAVOMavlinkBridge
+#OPTMODULES += UAVOMavlinkBridge
 OPTMODULES += UAVOMSPBridge
-OPTMODULES += UAVOLighttelemetryBridge
+#OPTMODULES += UAVOLighttelemetryBridge
 OPTMODULES += Battery
 OPTMODULES += ComUsbBridge
 OPTMODULES += Airspeed
-OPTMODULES += UAVOHoTTBridge
+#OPTMODULES += UAVOHoTTBridge
 OPTMODULES += UAVOFrSKYSensorHubBridge
```

Comment:
Will want to remove all these from HwSprf3e too https://github.com/d-ronin/dRonin/blob/next/shared/uavobjectdefinition/hwsprf3e.xml